website/js/ladder.js

```diff
@@ -26,7 +26,23 @@ var LadderPanel = Panels.StaticPanel.extend({
 	events: {
 		'change select[name=standing]': 'changeStanding',
 		'click button[name=openReset]': 'openReset',
-		'click button[name=cancelReset]': 'cancelReset'
+		'click button[name=cancelReset]': 'cancelReset',
+		'click button[name=copyUrl]': 'copyUrl',
```

Comment:
```suggestion
		'click button[name=copyUrl]': 'copyUrl'
```

We don't have eslint in `website/` but that's no reason to break ES3 compatibility!

Reply:
this does what now
is it really _that_ frail?

Reply:
smh ES3